cps/kobo.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```

Comment:
Could you please add here and in kobo_auth.py the GPL3 copyright text like in the other py files

Reply:
Done.